### PR TITLE
:art: chore/billing-hooks

### PR DESCRIPTION
**Pull request by vmcodes**
Pull-Request for `paretOS`

## Description
- Refactored BillingForm.tsx and Order.tsx to support hooks
- Made 'expId' optional in createExperience.ts
- Updated README.md to include Ubuntu issue

## Relates to
- #183

## Reviewers
- @mikhael28

### CODE REVIEW

**Review by mikhael28 — CHANGES_REQUESTED** [review]
😭 sorry to ask this, but can you please resolve the merge conflicts in `createExperience.ts`? Github says they are 'too complicated' to be resolved in the web editor. In general, it actually looks like we have a listing misconfiguration - can you disable Prettier, and just use the standard es-lint format? We recently updated the configuration, and I think we have removed the need for Prettier altogether. Until then, your code will keep having linting issues.

Also, please make sure to pull the latest code as usual 👍

**Comment by vmcodes** [comment]
@mikhael28 reviewing now, sorry for the late response

**Comment by mikhael28** [comment]
@vmcodes ping me when you resolve the merge conflicts :)

**Comment by vmcodes** [comment]
@mikhael28 hey Misha. I think I'm up to date now with the merge conflicts, but my most recent commit failed one of the checks. Can you review when possible?

**Comment by jayeclark** [comment]
Hey @vmcodes actually all of the integration tests have been failing recently because of a problem unrelated to contributors’ PRs. (I’ve been too busy between my new job & studying for AWS Certification to get around to debugging it yet… but hopefully soon I will be back. 😄)

TLDR: the integration tests will fail no matter what right now because of a config issue.

**Comment by vmcodes** [comment]
@jayeclark thank you for the update! :rocket:

**Review by mikhael28 — APPROVED** [review]
Approved! Thank you for adding the notes for installing on Ubuntu! Welcome back :)
